edocumentation-service/src/main/java/org/dcsa/edocumentation/service/mapping/ShippingInstructionMapper.java

```diff
@@ -2,18 +2,31 @@
 
 import org.dcsa.edocumentation.domain.persistence.entity.Shipment;
 import org.dcsa.edocumentation.domain.persistence.entity.ShippingInstruction;
+import org.dcsa.edocumentation.transferobjects.ShippingInstructionRefStatusTO;
 import org.dcsa.edocumentation.transferobjects.ShippingInstructionSummaryTO;
+import org.dcsa.edocumentation.transferobjects.ShippingInstructionTO;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 
-@Mapper(componentModel = "spring")
+@Mapper(componentModel = "spring",
+  uses = {
+    LocationMapper.class,
+  })
 public interface ShippingInstructionMapper {
   @Mapping(
     source = "shipments",
     target = "carrierBookingReferences")
   ShippingInstructionSummaryTO ShippingInstructionToShippingInstructionSummary(ShippingInstruction shippingInstruction);
```

Comment:
I think we need to make a choice to either separate the ShippingInstructionMapper and ShippingInstructionSummaryMapper or consolidate the mappers into one mapping class and optionally also combine (or leave separate) the services (ShippingInstructionService and ShippingInstructionSummaryService)

Reply:
BKG "cleanly" separates the two through and through. I will match that so we are consistent.

Reply:
Turns out the method in `ShippingInstructionMapper` was unused.